admin/templates/configuration/etc/qserv-czar.cnf

```diff
@@ -23,6 +23,8 @@ unix_socket = {{MYSQLD_SOCK}}
 host =
 user= {{MYSQLD_USER_QSERV}}
 port = 0
+# maximum user query result size in MB
+maxtablesize = 5500
```

Comment:
Even though the comment indicates this parameter's units are MB, the usual practice is to either include '_mb' into the name of the parameter:

  maxtablesize_mb = 5500

or to require units to be provided with the numbers:

  maxtablesize = 5500000000  # bytes
  maxtablesize = 5500000  # KB
  maxtablesize = 5500MB
  maxtablesize = 5GB
  ...

My personal choice would be the last one. Unfortunately this will require you to fix some of your code to avoid using "MB" in the names of variables.